p4-16/spec/P4-16-spec.mdk

```diff
@@ -6388,19 +6388,68 @@ annotations
 annotation
     : '@' name
     | '@' name '(' expressionList ')'
+    | '@' name '(' kvList ')'
```

Comment:
I hope we can use keyValueList instead of kvList.

Reply:
Done